test/system/inspect-provenance.test.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 import { test } from 'tap';
-import { chdirWorkspaces, activateVirtualenv } from './test-utils';
+import { chdirWorkspaces, activateVirtualenv } from '../test-utils';
 
-import pluginImpl = require('../lib');
+import pluginImpl = require('../../lib');
```

Comment:
I think that replacing `import pluginImpl = require('../../lib');` with `import { inspect } from '../../lib';` and instead of `pluginImpl.inspect` use `inspect` would do the job

Reply:
Loki will take care of it.